src/languages/internal/languagesconfiguration.cpp

```diff
@@ -34,14 +34,13 @@ using namespace mu::languages;
 
 static const Settings::Key LANGUAGE_KEY("languages", "language");
 
-static const QString LANGUAGES_SERVER_URL("http://extensions.musescore.org/4.0/languages/");
+static const QString LANGUAGES_SERVER_URL("http://extensions.musescore.org/4.1/languages/");
```

Comment:
Would it be possible to switch to https?

Reply:
Indeed, it looks like the site does not support https. fixing that would be out of scope for this PR.

Reply:
done, see #17712